### ISSUE TITLE
Obtain `1ES` exception for `CodeQL`/`ESLint` enforcement

### ISSUE THREAD

**Issue by scbedd** [Azure/azure-sdk-tools#7815]
[Example codeQL Timeout](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3482387&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=ce3b9047-3f2a-5c21-9248-60a34a33907c)

This task is on hold until this official process is rolled out,

> Once you add [justification](https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/features/sdlanalysis/disablingtools#use-justifications-to-disable-a-tool-using-sdl-exceptions), 1ES PT disables the corresponding tool immediately to unblock your pipeline. 1ES PT is currently working on enabling the following process, which will be rolled out early February 2024.

> - Owning service gets notified of SDL Exceptions via S360​ including the justification provided.
> - Business stakeholder (VP or above) needs to sign-off the S360 item within 48 hours with a grace period.
> - If sign-off does not happen or grace period expires, 1ES PT will break the pipeline.
> - Once SDL tool issue is mitigated, remove the yaml snippet (both enabled and justificationForDisabling) to re-enable the tool and close the S360 item.

This is still not present, reaching out to 1ES folks to confirm.

**Comment by scbedd** [comment]
[Asked 1ES this question](https://teams.microsoft.com/l/message/19:9ab85381899f49b3999d3a8d2a0aef4b@thread.tacv2/1711140918997?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=886b1d22-e648-4492-af5f-6fbe81e73d4a&parentMessageId=1711140918997&teamName=1ES%20Pipeline%20Templates%20(1ES%20PT)&channelName=Support%20-%20Onboarding%20to%201ES%20PT&createdTime=1711140918997)
